2-api/limits.md

```diff
@@ -2,6 +2,10 @@
 
 # Limits and errors
 
+The Perspective API uses limits and quotas to protect the system from receiving more data than it can handle, and to ensure equitable distribution of the system resources. These limits and quotas are subject to change.
+
+You may encounter an error if the quota of your request exceeds the limit.
```

Comment:
Rephrase to "You may encounter an error if your request exceeds the limit."

Should we be more specific about what the error messages are, if possible?